bin/migrate-mongo.js

```diff
@@ -102,19 +106,20 @@ program
   .command("status")
   .description("print the changelog of the database")
   .option("-f --file <file>", "use a custom config file")
-  .action(options => {
+  .action(async options => {
     global.options = options;
-    migrateMongo.database
-      .connect()
-      .then(({db, client}) => migrateMongo.status(db, client))
-      .then(statusItems => printStatusTable(statusItems))
-      .then(() => {
-        process.exit(0);
-      })
-      .catch(err => {
-        handleError(err);
-      });
-  });
+    try {
+      const { db } = await migrateMongo.database.connect();
+      const config = await migrateMongo.config.read();
```

Comment:
We need to read the config to see if we need to pass true to the `shouldLoadFromDatabase` parameter in the `status` function